### PR TITLE
When the Epicbox server settings are updated update secureStore _epic…

### PR DESCRIPTION
**Pull request by Likho**
…boxConfig

### CODE REVIEW

File: lib/pages/settings_views/epicbox_settings_view/epicbox_settings_view.dart [hunk 1]
```diff
@@ -51,28 +54,28 @@ class _WalletEpicBoxSettingsViewState
...
-          //         );
-          //       },
-          //       icon: const PlusIcon(
-          //         width: 20,
-          //         height: 20,
-          //       ),
-          //     ),
-          //   ),
-          // ],
+          actions: [
```
Comment (sneurlax) [inline lib/pages/settings_views/epicbox_settings_view/epicbox_settings_view.dart:57]: I think this section needs removing as we want to remove custom Epic Box server management for the time being

File: lib/utilities/default_epicboxes.dart [hunk 2]
```diff
@@ -4,8 +4,8 @@ abstract class DefaultEpicBoxes {
   static const String defaultName = "Default";
 
   static List<EpicBoxServerModel> get all =>
-      [americas /*, asia, europe*/]; // Temporarily disable non-americas regions
-  static List<String> get defaultIds => ['americas' /*, 'asia', 'europe'*/];
+      [americas, asia, europe]; // Temporarily disable non-americas regions
```
Comment (sneurlax) [inline lib/utilities/default_epicboxes.dart:7]: I think the asian and europe regional servers needs removing due to recently wanting only the americas server available

**Comment by Likho** [comment]
Closing PR for now, we can reopen later when we want to enable all the epicbox servers

File: lib/services/coins/epiccash/epiccash_wallet.dart [hunk 3]
```diff
@@ -1175,24 +1176,30 @@ class EpicCashWallet extends CoinServiceAPI {
...
-    // EpicBoxServerModel? _epicBox = epicBox ??
-    //     DB.instance.get<EpicBoxServerModel>(
-    //         boxName: DB.boxNamePrimaryEpicBox, key: 'primary');
-    // Logging.instance.log(
-    //     "Read primary Epic Box config: ${jsonEncode(_epicBox)}",
-    //     level: LogLevel.Info);
-
     if (storedConfig == null) {
-      // if no config stored, use the default epicbox server as config
+      // if no config stored, use the default epicbox server as config, this is on first creation/ restore
```
Comment (sneurlax) [inline lib/services/coins/epiccash/epiccash_wallet.dart:1180]: This whole section ought to be merged into main, it's a good change

**Pull request closed without merging**